templates/telco/openran.html

```diff
@@ -0,0 +1,490 @@
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/6c534a44-oran.png",
+              alt="",
+              width="1500",
+              height="643",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/176a11dd-opensource-MANO.svg",
+              alt="",
+              width="300",
+              height="108",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f253ee6e-telecom-infra-project-vector-logo.svg",
+              alt="",
+              width="150",
+              height="150",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/314085ad-openstack.svg",
+              alt="",
+              width="300",
+              height="78",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/9f8a8273-2018-logo-kubernetes+%282%29.svg",
+              alt="",
+              width="356",
+              height="60",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/04115a7e-ubuntu_black-orange_hex.svg",
+              alt="",
+              width="300",
+              height="67",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/038514e8-MAAS_black-orange_rgb.svg",
+              alt="",
+              width="300",
+              height="79",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/48a8eddf-juju_black-orange_hex.svg",
+              alt="",
+              width="407",
+              height="147",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
```

Comment:
This section of logos doesn't exist in the copy docs?

Reply:
Those are logos found in the image that is in the copydoc